scripts/build

```diff
@@ -106,12 +112,12 @@ if [ ! -x ${INSTALLBIN}/cni ]; then
     WORKDIR=$TMPDIR/src/github.com/containernetworking/plugins
     git clone -b $VERSION_CNIPLUGINS https://github.com/rancher/plugins.git $WORKDIR
     cd $WORKDIR
-    GO111MODULE=off GOPATH=$TMPDIR CGO_ENABLED=0 "${GO}" build -tags "$TAGS" -ldflags "$VERSIONFLAGS $LDFLAGS $STATIC" -o $INSTALLBIN/cni
+    GO111MODULE=off GOPATH=$TMPDIR CGO_ENABLED=0 "${GO}" build -tags "$TAGS" -gcflags="all=${GCFLAGS}" -ldflags "$VERSIONFLAGS $LDFLAGS $STATIC" -o $INSTALLBIN/cni
```

Comment:
If debug isn't set this ends up setting `-gcflags="all="` - does that change the build behavior?

Reply:
In my understanding no, as the default is equivalent to passing no flags:

https://github.com/golang/go/blob/638c9aad5f88f96d9aa525bbe403c8a5d3b743e8/src/cmd/dist/build.go#L878-L880

This reflects what I have observed in manual tests (equivalent binaries).